docs/release-notes/index.md

```diff
@@ -21,6 +21,261 @@ To check for security updates, go to [Security announcements for the Elastic sta
+* Prevents Lens Embeddable `defaultTitle` from being overwritten with a custom title after reload [#225664]({{kib-pull}}225664).
+* Fixes panel title synchronization with corresponding saved object when using `defaultTitle` [#225237]({{kib-pull}}225237).
+* Fixes visual issues causing labels to be truncated [#225430]({{kib-pull}}225430).
+* Refreshes "Values from a query" options for {{esql}} controls on dashboard reload [#225101]({{kib-pull}}225101).
+* Fixes an issue with calculating the query for retrieving {{esql}} control values [#214905]({{kib-pull}}214905).
+* Fixes an issue with the {{esql}} **Create control** suggestions not triggering if the query already contained a control [#214833]({{kib-pull}}214833).
+* Fixes the visibility of the date picker when writing {{esql}} visualization queries [#214728]({{kib-pull}}214728).
+* Fixes a performance issue with {{esql}} visualizations in case of errors in the query [#225067]({{kib-pull}}225067).
+* Fixes dashboard control value changes causing multiple fetches [#224761]({{kib-pull}}224761).
+* Fixes an issue in **Lens** where reordering the groups within a layer would incorrectly assign the color mapping to a group other than the first [#215426]({{kib-pull}}215426).
+* Fixes invalid dashboard displayed as 404 instead of showing validation error [#211661]({{kib-pull}}211661).
+* Fixes an issue where custom ranges and multi-field values were not correctly colored based on selected color mapping configurations [#207957]({{kib-pull}}207957).
+
+**Data ingestion and Fleet**:
+* Fixes CSV export in the agent list [#225050]({{kib-pull}}225050).
+* Replaces registry call when deleting {{kib}} assets for custom packages [#224886]({{kib-pull}}224886).
+* Ensures package policy names are unique when moving across spaces [#224804]({{kib-pull}}224804).
+* Fixes bulk actions incorrectly selecting agents when a namespace filter is applied [#224036]({{kib-pull}}224036).
+* Reverts "Added instructions for installing {{agent}} complete." [#223520]({{kib-pull}}223520).
+* Fixes styled component theme lookup issue [#221979]({{kib-pull}}221979).
+* Fixes SSL config being overridden by advanced YAML in full agent policy [#219902]({{kib-pull}}219902).
+* Fixes required capability for the SIEM migrations topic [#219427]({{kib-pull}}219427).
+* Makes output and {{fleet}} Server settings non-editable for agentless policies [#218905]({{kib-pull}}218905).
+* Supports integrations with secrets that contain multiple values [#216918]({{kib-pull}}216918).
+* Adds remote cluster instructions for syncing integrations [#211997]({{kib-pull}}211997).
+* Updates install snippets to include all platforms [#210249]({{kib-pull}}210249).
+
+**Discover**:
+* Fixes invalid input highlight in **Data View** flyout [#226822]({{kib-pull}}226822).
+* Fixes an issue causing an internal server error when renaming a search session [#226757]({{kib-pull}}226757).
```

Comment:
This one was a dev mode only bug, not sure if we usually include those kinds of things or not, so just mentioning.

Reply:
```suggestion
```